wagtail/admin/templates/wagtailadmin/edit_handlers/inline_panel_child.html

```diff
@@ -11,10 +11,10 @@
     {% endif %}
     <ul class="controls">
         {% if can_order %}
-            <li><button type="button" class="button icon text-replace white icon-order-up inline-child-move-up" id="{{ child.form.prefix }}-move-up" title="{% trans 'Move up' %}">{% trans "Move up" %}</button></li>
-            <li><button type="button" class="button icon text-replace white icon-order-down inline-child-move-down" id="{{ child.form.prefix }}-move-down" title="{% trans 'Move down' %}">{% trans "Move down" %}</button></li>
+            <li><button type="button" class="button button--icon text-replace white inline-child-move-up" id="{{ child.form.prefix }}-move-up" title="{% trans 'Move up' %}">{% icon name="order-up" %}{% trans "Move up" %}</button></li>
```

Comment:
Reminder: add `button-small` to restore previous functionality. There was a specificity issue that meant this was getting the small button styles